acceptance_tests/features/steps/rh_ui.py

```diff
@@ -83,3 +86,15 @@ def error_section_displayed(context, href_name, expected_text):
     test_helper.assertEqual(context.browser.find_by_id('alert').text, 'There is a problem with this page')
     error_text = context.browser.links.find_by_href(href_name).text
     test_helper.assertEqual(error_text, expected_text)
+
+
+@step("check UAC is in firestore via eqLaunched endpoint")
```

Comment:
Where possible steps should describe the actions from the users point of view, this could be worded like 
`"the UAC is valid in RH UI"`
💅